ios/RCCViewController.m

```diff
@@ -814,4 +845,32 @@ -(BOOL)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer shouldRecogniz
   return !disabledSimultaneousGestureBool;
 }
 
+#pragma mark - UIViewControllerPreviewingDelegate
+- (UIViewController *)previewingContext:(id<UIViewControllerPreviewing>)previewingContext viewControllerForLocation:(CGPoint)location {
+  return self.previewController;
+}
+
+- (void)previewingContext:(id<UIViewControllerPreviewing>)previewingContext commitViewController:(UIViewController *)viewControllerToCommit {
+  [self.navigationController pushViewController:self.previewController animated:false];
```

Comment:
Here we may want to provide the user to skip this operation if he doesn't want the screen to be pushed.